src/main/cpp/faultinj/README.md

```diff
@@ -0,0 +1,98 @@
+# Fault Injection Tool for CUDA Runtime and CUDA Driver API
+
+The goal of this tool is to increase testability of the failure handling logic
```

Comment:
Brilliant work here. Thank you for documenting this.

Reply:
thank you!